defaults/main.yml

```diff
@@ -10,6 +10,7 @@ foreman_scap_client_host_cert_path: ''
 foreman_scap_client_host_private_key_path: ''
 foreman_scap_client_release: latest
 foreman_scap_client_repo_url: 'https://yum.theforeman.org/client/{{ foreman_scap_client_release }}/el7/x86_64'
+foreman_scap_client_apt_key_url: 'https://apt.atix.de/atix_gpg.pub'
 foreman_scap_client_apt_repo_url: 'deb [trusted=yes] https://apt.atix.de/Ubuntu20LTS/ stable main'
```

Comment:
Maybe we should adapt this line of code. Don't like this hardcoded value to pick Ubuntu20LTS and then also add trusted=yes. It's a little bit weird.